### PR TITLE
Feature/border radius dropdowns

### PR DESCRIPTION
**Pull request by xmurcia**
## Description

Add a couple of variables to style the dropdown border radius.

## Related Issue

[MTR-50188](https://jira.scmspain.com/browse/MTR-50188)

### CODE REVIEW

**Review by tomcask — APPROVED** [review]

File: packages/sui-theme/src/settings-compat-v7/_components.scss [hunk 1]
```diff
@@ -182,21 +182,23 @@ $size-dropdown-icon: 18px !default;
 $z-dropdown-menu: 1 !default;
 
 // Dropdown basic
-$fw-dropdown-basic-button: inherit !default;
+$bdr-dropdown-basic-menu: 0 !default;
```
Comment (turolopezsanabria) [inline packages/sui-theme/src/settings-compat-v7/_components.scss:185]:
```suggestion
$bdrs-dropdown-basic-menu: 0 !default;
```

File: packages/sui-theme/src/settings-compat-v7/_components.scss [hunk 2]
```diff
@@ -182,21 +182,23 @@ $size-dropdown-icon: 18px !default;
...
 $c-dropdown-basic-button-hover: $c-accent !default;
 $c-dropdown-basic-button-icon: $c-accent !default;
 $c-dropdown-basic-list-link: $c-accent !default;
+$fw-dropdown-basic-button: inherit !default;
 
 // Dropdown user
 $bd-dropdown-user-last-item: 1px solid $c-gray-light !default;
+$bdr-dropdown-user-menu: 0 !default;
```
Comment (turolopezsanabria) [inline packages/sui-theme/src/settings-compat-v7/_components.scss:193]:
```suggestion
$bdrs-dropdown-user-menu: 0 !default;
```

Comment (turolopezsanabria) [reply packages/sui-theme/src/settings-compat-v7/_components.scss:193]: `$bdrs` for broder radius (`$bdr` is border right)

**Review by turolopezsanabria — CHANGES_REQUESTED** [review]

File: packages/sui-theme/src/settings-compat-v7/_components.scss (hunk 2 above)
Comment (xmurcia) [reply packages/sui-theme/src/settings-compat-v7/_components.scss:193]: Ok! Thanks!

**Review by kikoruiz — APPROVED** [review]

**Review by turolopezsanabria — APPROVED** [review]
merci!

**Pull request merged**